src/Editor/HoverProvider.ts

```diff
@@ -28,7 +29,15 @@ export class HoverProvider implements vscode.HoverProvider {
       return new vscode.Hover(mdfile);
     }
     const word = _doc.getText(range);
-    // TODO Add more
+
+    toolsAttr.forEach((item) => {
+      if (item.name === word) {
+        mdfile.appendMarkdown(`### ${item.name}\n`);
+        mdfile.appendMarkdown(`${item.description}\n`);
```

Comment:
I didn't notice `let mdfile = new vscode.MarkdownString();` in previouse PR,
why is the variable name have file? is this really a file?

Reply:
Oh, it's not a file.
I think the name of the variable will be confusing, so can I change it to `markdownString`?

Reply:
>  so can I change it to markdownString?

Yes, that seems nice... but in another PR :)

Reply:
I'll post a new PR!

Reply:
I posted a new PR #201

Do I just leave the current PR without closing it?

Reply:
> Do I just leave the current PR without closing it?

When #201 lands, you can rebase this PR to `main` branch.

Reply:
Yes, I'll do that. Thank you.

Reply:
I revised it!